tests/metrics/genai/test_genai_metrics.py

```diff
@@ -769,6 +769,14 @@ def test_faithfulness_metric():
             examples=[mlflow_example],
         )
 
+    faithfulness_metric.eval_fn(
+        # Inputs with different indices
```

Comment:
nit: the error is rather because of index not starting from 0 I guess?
```
for indx, (input, output) in enumerate(zip(inputs, outputs)):
            try:
                arg_string = _format_args_string(grading_context_columns, eval_values, indx)
```

The `indx` always starts from 0 (cuz we do enumerate) and `eval_values[0]` will raise KeyError if it doesn't have index 0.

Reply:
Exactly, `reset_index(drop=True)` should also work.

Reply:
yup I think the current solution is fine, just wanted to make sure the ^comment is accurate:)